inspector/src/components/persistentPopupHostComponent.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { PopupComponent } from "./popupComponent";
+import { Nullable } from "babylonjs/types";
+import { GlobalState } from "./globalState";
+import { Observer } from "babylonjs/Misc/observable";
+
+interface IPersistentPopupHostComponentProps {
+    globalState?: GlobalState;

```

Comment:
this should also not be optional. Otherwise this component cannot function. it is bound to the global state.

You can also use a part of the global state (i.e. the part you need), but since this is currently only used in the inspector and not shared, there is no need to decouple it from the inspector's global state.